### PR TITLE
updates to Agility CMS info

### PR DESCRIPTION
**Pull request by jamesvidler**
## Description
Added the Category **CMS: Agility CMS** for starter sites. Also updated the starter docs for Agility CMS, and info regarding the Agility CMS source plugin.

### CODE REVIEW

File: docs/docs/sourcing-from-agilitycms.md [hunk 1]
```diff
@@ -4,13 +4,19 @@ title: Sourcing from Agility CMS
...
+Check our [docs](https://help.agilitycms.com/hc/en-us/articles/360039879872) for up-to-date documentation.
+
 ## What is Agility CMS? What makes it different?
 
-[Agility CMS](https://agilitycms.com/) is a headless Content Management System (CMS) that lets you define your custom content types and relationships. This is called Content Architecture, and you can reuse this content for your websites and apps.
+[Agility CMS](https://agilitycms.com/) is a headless Content Management System (CMS) that lets you define your custom content types, relationships and pages. This is called Content Architecture, and you can reuse this content for your websites and apps.
+
+Agility believes that a successful website balances the **User Experience (UX)**, **Editor Experience (EX)**, and **Developer Experience (DX)**. 
+
+While Gatsby tends to handle **UX** and **DX** quite well, too often editors are an after-thought in Gatsby (and other JAMstack) websites. They feel constrained by not being able to manage their sitemap and what content is on which pages.
```
Comment (laurieontech) [inline docs/docs/sourcing-from-agilitycms.md]: I might suggest some rephrasing here. This is a potential option.
```suggestion
While Gatsby tends to handle **UX** and **DX** quite well, editors are not always comfortable working directly in a codebase. They prefer to manage their sitemap and see what content is on which pages. Using Gatsby with a headless CMS allows for this.
```

**Comment by laurieontech** [comment]
Thanks for the changes! Looks like your branch is protected. Can you run `yarn format` and push the changes? That should fix the linter test.

**Comment by jamesvidler** [comment]
@laurieontech thanks for the help! I ran `yarn format`, did resolve it?

**Comment by laurieontech** [comment]
Yup! Waiting for the full tests to run, but hopefully this will be good to go.

File: docs/categories.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ starter:
   - AWS
   - Blog
   - Client-side App
+  - CMS:Agility CMS
```
Comment (laurieontech) [inline docs/categories.yml]:
```suggestion
  - CMS:Agility
```

File: docs/starters.yml [hunk 3]
```diff
@@ -3793,11 +3793,11 @@
     - Styled Components
     - Sitemap Generation
     - Google Analytics
-- url: https://agility-gatsby-starter.netlify.com
+- url: https://agility-gatsby-starter-gatsbycloud.netlify.com
   repo: https://github.com/agility/agility-gatsby-starter
   description: Get started with Gatsby and Agility CMS using a minimal blog.
   tags:
-    - CMS:Other
+    - CMS:Agility CMS
```
Comment (laurieontech) [inline docs/starters.yml]:
```suggestion
    - CMS:Agility
```

Comment (jamesvidler) [reply docs/starters.yml]: I agree it does sound redundant, but **Agility CMS** is our name and we need to use this format for SEO. There are other companies named **Agility** (not related to CMS) so it's important that we are consistent in how we use our company name publicly.

**Comment by laurieontech** [comment]
Understood. Appreciate the context!

**Pull request merged**